File: URI Programs/C/1040.c
#include <stdio.h>

int main(){
	float n1, n2, n3, n4, average, exam;
	scanf("%f %f %f %f", &n1, &n2, &n3, &n4);
	average = (n1 * 2 + n2 * 3 + n3 * 4 + n4 * 1) / 10;
	printf("Media: %.1f\n", average);
	if (average >= 7){
		printf("Aluno aprovado.\n");
	}
	else if (average >= 5){
		printf("Aluno em exame.\n");
		scanf("%f", &exam);
		printf("Nota do exame: %.1f\n", exam);
		average = (average + exam) / 2;
		if (average >= 5){
			printf("Aluno aprovado.\n");
		} else {
			printf("Aluno reprovado.\n");
		}
		printf("Media final: %.1f\n", average);
	}
	else {
		printf("Aluno reprovado.\n");
	}
	return 0;
}

File: URI Programs/Python/1040.py
N1 , N2 , N3 , N4 = map(float, input().split())

MED = ((N1*2) + (N2*3) + (N3*4) + (N4*1))/10

print("Media: "+str("%.1f" %MED))

if MED < 5.0:
  print("Aluno reprovado.")
elif MED >= 7.0:
  print("Aluno aprovado.")
elif 5.0 <= MED < 7.0:
 print("Aluno em exame.") 
 N5 = float(input())
 print("Nota do exame: "+str(N5))
 Med = (MED+N5)/2 
 if Med >= 5.0:
   print("Aluno aprovado.")
   print("Media final: "+str("%.1f" %Med))
 elif Med < 5.0:
    print("Aluno reprovado.")
    print("Media final: "+str("%.1f" %Med))